### ISSUE TITLE
Translate all strings for `Improved UX for agency card enrollment`

### ISSUE THREAD

**Issue by indexing** [cal-itp/benefits#2075]
This issue addresses Spanish translations for the copy changes in this milestone: https://github.com/cal-itp/benefits/milestone/45

## Acceptance Criteria

- [x] @indexing submits new strings to the translation service
- [x] @indexing receives translated strings back from translation service
- [x] @indexing verifies translations
- [x] engineering implements in the PO files

**Comment by indexing** [comment]
I've added Spanish translations for the updates to our agency card enrollment pathway to [Cal-ITP Benefits Application Copy](https://docs.google.com/spreadsheets/d/1_Gi_YbJr4ZuXCOsnOWaewvHqUO1nC1nKqiVDHvw0118/edit?usp=sharing). See the forTranslation sheet; all new strings are highlighted in yellow.

For future reference, and to provide an easier option for adding translations for these few views, I've attached the CSV file that includes the error state strings. (The file also includes those for error states, but perhaps the smaller subset will make updating the PO files quicker.)

[Spanish-UX Improvements Error States05.01.2024.csv](https://github.com/cal-itp/benefits/files/15201737/Spanish-UX.Improvements.Error.States05.01.2024.csv)

cc// [@thekaveman](https://github.com/thekaveman)
